app/approve/registrations/approve.registrations.controller.js

```diff
@@ -11,10 +11,7 @@
         const vm = this;
 
         let user = UserService.get();
-
-        vm.usersList = [];
-        vm.currentUser = {};
-        vm.adminPlatoonName = user.platoon.name;
+        let userAuthority = UserService.getTopAuthority().id;
```

Comment:
Will be better if you assign to `userAuthority` the authority object and then use this object and his `id`